src/model/series/pane-view.ts

```diff
@@ -1,12 +1,16 @@
 import { IUpdatablePaneView } from '../../views/pane/iupdatable-pane-view';
 
 import { IChartModelBase } from '../chart-model';
-import { CustomData, CustomSeriesPricePlotValues, CustomSeriesWhitespaceData } from '../icustom-series';
+import { CustomConflationContext, CustomData, CustomSeriesPricePlotValues, CustomSeriesWhitespaceData } from '../icustom-series';
 import { ISeries } from '../iseries';
 import { SeriesType } from '../series-options';
 
-export interface ISeriesCustomPaneView extends IUpdatablePaneView {
-	priceValueBuilder(plotRow: CustomData<unknown> | CustomSeriesWhitespaceData<unknown>): CustomSeriesPricePlotValues;
-	isWhitespace(data: CustomData<unknown> | CustomSeriesWhitespaceData<unknown>): data is CustomSeriesWhitespaceData<unknown>;
+export interface ISeriesCustomPaneView<
+	HorzScaleItem = unknown,
+	TData extends CustomData<HorzScaleItem> = CustomData<HorzScaleItem>
+> extends IUpdatablePaneView {
+	priceValueBuilder(plotRow: TData | CustomSeriesWhitespaceData<HorzScaleItem>): CustomSeriesPricePlotValues;
+	isWhitespace(data: TData | CustomSeriesWhitespaceData<HorzScaleItem>): data is CustomSeriesWhitespaceData<HorzScaleItem>;
+	conflationReducer?(items: readonly CustomConflationContext<HorzScaleItem, TData>[]): TData;
```

Comment:
```suggestion
	conflationReducer: undefined | ((items: readonly CustomConflationContext<HorzScaleItem, TData>[]) => TData);
```
Make the method exist but possibly return undefined. This simplifies other code.